endpointmanager/pkg/chplendpointquerier/NextGenwebscraper.go

```diff
@@ -21,21 +21,19 @@ func NextGenwebscraper(CHPLURL string, fileToWriteTo string) {
 	doc.Find("table").Each(func(index int, tablehtml *goquery.Selection) {
 		tablehtml.Find("tbody").Each(func(indextr int, rowhtml *goquery.Selection) {
 			rowhtml.Find("tr").Each(func(indextr int, rowbodyhtml *goquery.Selection) {
-				var entryDSTU2 LanternEntry
-				var entryR4 LanternEntry
+				var entry LanternEntry
+
 				tableEntries := rowbodyhtml.Find("td")
 				if tableEntries.Length() > 0 {
 					organizationName := strings.TrimSpace(tableEntries.Eq(1).Text())
-					DSTU2URL := strings.TrimSpace(tableEntries.Eq(6).Text())
-					R4URL := strings.TrimSpace(tableEntries.Eq(7).Text())
-
-					entryDSTU2.OrganizationName = organizationName
-					entryDSTU2.URL = DSTU2URL
+					zipCode := strings.TrimSpace(tableEntries.Eq(5).Text())
+					URL := strings.TrimSpace(tableEntries.Eq(6).Text())
 
-					entryR4.OrganizationName = organizationName
-					entryR4.URL = R4URL
+					entry.OrganizationName = organizationName
+					entry.URL = URL
+					entry.OrganizationZipCode = zipCode
```

Comment:
It looks like **some** of the NextGen zip code is 9 characters, do we cut the zipcodes down to 5 characters here or somewhere else?

Reply:
I didn't notice this! I just added a check to the Lantern format fetcher parser that will cut down any zip codes longer than 5 characters down to 5!